src/main/java/com/lubycon/devti/domain/bucket_test_type/entity/BucketTestType.java

```diff
@@ -0,0 +1,42 @@
+package com.lubycon.devti.domain.bucket_test_type.entity;
+
+import com.lubycon.devti.global.code.TestType;
+import com.lubycon.devti.global.entity.BaseTimeEntity;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(AccessLevel.PROTECTED)
```

외부에서 기본 생성자 생성을 막기 위해 접근 권한을 지정해줘도 좋을 것 같아요!

Reply:
앗 선영님 코드보고 적용해보려했는데 까먹고 안했네요! 감사합니다!!